WorldBank-Transport/DRIVER
# Add created_by to record list

## Overview

Add "Created by" field to the record list.

### Checklist


### Screenshots

![record_list_created_by](https://user-images.githubusercontent.com/2926237/45243042-5e475c00-b2c0-11e8-89d6-07454d64bad6.png)

## Testing Instructions

* Go to http://localhost:7000/#!/list and make sure the field shows up with appropriate values

Closes #159626646
